docs/security/hardening-guide.md

```diff
@@ -36,13 +39,57 @@ kernel.keys.root_maxbytes=25000000
 
 ## Kubernetes Runtime Requirements
 
-The runtime requirements to comply with the CIS Benchmark are centered around pod security (PSPs), network policies and API Server auditing logs. These are outlined in this section. K3s doesn't apply any default PSPs or network policies. However, K3s ships with a controller that is meant to apply a given set of network policies. By default, K3s runs with the `NodeRestriction` admission controller. To enable PSPs, add the following to the K3s start command: `--kube-apiserver-arg="enable-admission-plugins=NodeRestriction,PodSecurityPolicy,ServiceAccount"`. This will have the effect of maintaining the `NodeRestriction` plugin as well as enabling the `PodSecurityPolicy`. The same happens with the API Server auditing logs, K3s doesn't enable them by default, so audit log configuration and audit policy must be created manually.
+The runtime requirements to comply with the CIS Benchmark are centered around pod security (via PSP or PSA), network policies and API Server auditing logs. These are outlined in this section.
+
+By default, K3s does not apply any pod security or network policies. However, K3s ships with a controller that is meant to apply a given set of network policies. The same setup applies API Server auditing logs, K3s doesn't enable them by default, so audit log configuration and audit policy must be created manually. By default, K3s runs with the `NodeRestriction` admission controller. 
```

Comment:
What controller do we have that applies network policies? Do you mean that it enforces the policies, if the user applies (creates) rules in the cluster?

```suggestion
By default, K3s does not include any pod security or network policies. However, K3s ships with a controller that will enforce network policies, if any are created. K3s doesn't enable auditing by default, so audit log configuration and audit policy must be created manually. By default, K3s runs with the both the `PodSecurity` and `NodeRestriction` admission controllers enabled, among others.
```

Reply:
My dumb answer is: I don't know, this is an paragraph from the existing documentation. As such I am totally fine with taking your suggestions as fact.